staff/meetings/bod

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+import argparse
+import meetings
+import re
+from datetime import datetime
+from collections import defaultdict
+from math import ceil
+
+
+def quorum():
+    print( meetings.quorum() )
+
+
+def ls():
+    d = meetings.ls()
+    for user in d:
+        print( user, d[ user ] )
+
+
+def danger():
+    path = meetings.get_current_minutes_path( "bod" ) 
+    minutes = meetings.get_minutes( path )
+    if len( minutes ) > 0:
+        bod = meetings.ls( state = "bod" )
+        a = meetings.attendance( path + "/" + minutes[-1] )
+        for user in [ user for user in bod if user not in a ]:
```

Comment:
i would just make these sets and write

```python
for user in bod - a:
```

(idk what `a` is, name it better?)